src/Runner.php

```diff
@@ -14,22 +14,50 @@ class Runner
 {
     use IO;
 
-    const VERSION = '0.5.4';
+    const VERSION = '0.6.0';
     const ROBOCLASS = 'RoboFile';
     const ROBOFILE = 'RoboFile.php';
-
+    
+    /**
+     * @var CurrentDir
```

Comment:
I just actually meant the `@var` declaration and not the usage, but you seem right. Those seem to be left-overs...